plexus-compiler-api/src/main/java/org/codehaus/plexus/compiler/CompilerConfiguration.java

```diff
@@ -354,16 +356,26 @@ public boolean isShowDeprecation()
         return showDeprecation;
     }
 
+    public String getWarnings()
+    {
+        return warnings;
+    }
+
+    public void setShowLint( String warnings )
```

Comment:
I find this lack of symmetry between `get`/`set`/`field` disturbing.
Also - other `setShow*`s seem to accept boolean.

Reply:
@pzygielo ah yes I agree It's confusing I will change that

Reply:
ah yes good point I agree

Reply:
@pzygielo @bondolo what about https://github.com/codehaus-plexus/plexus-compiler/pull/132
btw this fix the Xlint with no args which is possible

Reply:
The problem was that there was an existing showWarnings() flag. The intention was that the warnings flag would still control whether lint would be shown to avoid the weird corner case where warnings were disabled and lint was configured. 
warnings enabled : show errors, warnings and "lint"
warnings disabled : show errors only.